test/Microsoft.VisualStudio.Editor.Razor.Test/TestFiles/ParserTests/DefaultVisualStudioRazorParserIntegrationTest/AwaitPeriodInsertionAcceptedProvisionally.stree.txt

```diff
@@ -0,0 +1,19 @@
+RazorDocument - [0..20)::20 - [foo @await Html. baz]
```

Comment:
I've verified most of these tests manually. Should be enough to sanity check just a few.